whitesands/code/modules/clothing/head/jobs.dm

```diff
@@ -46,3 +46,23 @@
 	icon_state = "cowboyhos"
 
 	dog_fashion = /datum/dog_fashion/head/cowboy
+
+/obj/item/clothing/head/cowboy/sec/roumain
+	name = "hunter's hat"
+	desc = "A fancy hat with a nice feather. It covers your eyes, which makes you feel badass."
```

Comment:
desc = "A fancy hat with a nice feather. The way it covers your eyes makes you feel like a badass."